phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -76,6 +79,11 @@ public RepositoryDTO fetchInstallableApplications() {
         }
     }
 
+    @Override
+    public boolean isSafe() {
```

Comment:
This is a stub method for now. This concept of safe repository will be handled in a second time